### PR TITLE
[Dashboard De-Angular] Initial clean up and linter fix

### PR DESCRIPTION
**Pull request by abbyhu2000**
### Description

Fix linter and some clean ups.

### CODE REVIEW

**Review by ashwin-pc — COMMENTED** [review]
Are all the TODO's that you are removing resolved?

File: src/plugins/dashboard/public/application/utils/use/use_dashboard_container.tsx [hunk 1]
```diff
@@ -339,8 +335,6 @@ const createDashboardEmbeddable = (
             return Object.values(differences).length === 0 ? undefined : cloneDeep(differences);
           };
 
-          // TODO: handle dashboard container input and output subsciptions
```
Comment (abbyhu2000) [inline src/plugins/dashboard/public/application/utils/use/use_dashboard_container.tsx:342]: resolved

File: src/plugins/dashboard/public/application/utils/use/use_dashboard_container.tsx [hunk 2]
```diff
@@ -381,13 +375,6 @@ const createDashboardEmbeddable = (
             ) {
               // Add filters modifies the object passed to it, hence the clone deep.
               filterManager.addFilters(cloneDeep(container.getInput().filters));
-
-              // TODO: investigate if this is needed
```
Comment (abbyhu2000) [inline src/plugins/dashboard/public/application/utils/use/use_dashboard_container.tsx:385]: resolved, this is not needed here

File: src/plugins/dashboard/public/dashboard.ts [hunk 3]
```diff
@@ -90,8 +82,6 @@ export class Dashboard<TDashboardParams = DashboardParams> {
       this.description = state.description;
     }
     if (state.panels) {
-      // this panels is only JSON.parse() panels, we should convert them into the same type as app state panels
```
Comment (abbyhu2000) [inline src/plugins/dashboard/public/dashboard.ts:93]: resolve

**Comment by abbyhu2000** [comment]
> Are all the TODO's that you are removing resolved?

yes for the ones I deleted

**Review by ashwin-pc — APPROVED** [review]

**Pull request merged**